### PR TITLE
Add powertool tests into GitHub actions

### PR DESCRIPTION
**Pull request by maqiuyujoyce**
### Change description

Fix the testdata for the powertool tests and add it to the GitHub actions.

Blocked by the merge of #1891.

### Tests you have done

The following test run passed:

```
RUN_E2E=1 E2E_KUBE_TARGET=envtest E2E_GCP_TARGET=mock GOLDEN_REQUEST_CHECKS=1 \
  go test -test.count=1 -timeout 600s -v ./tests/e2e \
  -run TestE2EScript/scenarios/powertool
```

### CODE REVIEW

File: tests/e2e/testdata/scenarios/powertool/clear_state_into_spec/script.yaml
[the remark this answers is not in the document]
Comment (maqiuyujoyce) [reply tests/e2e/testdata/scenarios/powertool/clear_state_into_spec/script.yaml:1]: Done.

**Comment by yuwenma** [comment]
Could you rebase this PR (its diff-base is merged)?

**Comment by maqiuyujoyce** [comment]
> Could you rebase this PR (its diff-base is merged)?

It was already rebased yesterday!

**Comment by yuwenma** [comment]
/lgtm
/approve

Cool! Thanks

**Pull request merged**